agent/php_execute.c

```diff
@@ -1930,15 +1930,29 @@ void php_observer_handle_exception_hook(zval* exception, zval* exception_this) {
 
 static void nr_php_observer_attempt_call_cufa_handler(NR_EXECUTE_PROTO) {
   NR_UNUSED_FUNC_RETURN_VALUE;
+
+  if (NULL == execute_data || NULL == execute_data->opline) {
```

Comment:
It seems this checks are not needed. 
1. `execute_data` is guaranteed not to be `NULL`. `nr_php_observer_attempt_call_cufa_handler` is called [here](https://github.com/newrelic/newrelic-php-agent/blob/oapi/agent/php_execute.c#L2025) from `nr_php_instrument_func_begin`. `nr_php_instrument_func_begin` is called [here](https://github.com/newrelic/newrelic-php-agent/blob/oapi/agent/php_execute.c#L2285) from `nr_php_observer_fcall_begin` and `nr_php_observer_fcall_begin` ensures `execute_data` is not NULL [here](https://github.com/newrelic/newrelic-php-agent/blob/oapi/agent/php_execute.c#L2262).
2. `execute_data->opline` is not used anywhere in `nr_php_observer_attempt_call_cufa_handler`.

I guess maybe checking `execute_data->opline` makes sense to ensure that `execute_data` received by `nr_php_observer_attempt_call_cufa_handler` is not corrupt? But in case it tunrs out to be corrupt maybe agent should log something?

Reply:
previous conversation about this line here: https://github.com/newrelic/newrelic-php-agent/pull/559/files#r1003742842

Reply:
I'm ok with leaving it as is after b8052af32e58038789cff46e6054b8426cc13af2.